MapboxGeocoder/MBPlacemark.swift

```diff
@@ -125,9 +125,17 @@ open class Placemark: NSObject, Codable {
         }
     }
     
+    #if swift(>=4.2)
+    open override var hash: Int {
```

Comment:
You can most likely drop this hash property too.

Reply:
Oh, I believe you are right. However, I still need to manually create a hash for < swift 4.2 right as it isn't automatically synthesized?